install_config/install/deploy_router.adoc

```diff
@@ -230,6 +230,45 @@ endif::[]
 You can link:../../admin_guide/high_availability.html[set up a highly-available
 router] on your OpenShift cluster using IP failover.
 
+
+[[configuring-router-sharding]]
+=== Configuring Router Sharding
+
+Router sharding is
+link:../../architecture/core_concepts/routes.html#router-sharding[based on labels];
+you set labels on the router
+and make sure there is a corresponding node selector
+when enabling that router.
+This can be done with a single command, per shard.
+The following creates three shards, by geo.
+
+----
+$ router-shard latam
+$ router-shard emea
+$ router-shard apac
+----
+
+It uses the convenience script `router-shard`:
+
+====
+[source,bash]
+----
+#!/bin/bash
+geo=$1
+oadm router router-$geo --replicas=3 \
```

Comment:
Does this need a # at the start? Perhaps not, cos it's in a script-thing, but I thought I'd ask.